### DISCUSSION TITLE
A problem with Fake McAfee WebAdvisory Pop-up ?

### DISCUSSION THREAD

**Discussion by StefanOlovJansson** [wixtoolset/issues#7410] [Questions]
I downloaded the installer wix311.exe from https://github.com/wixtoolset/wix3/releases/tag/wix3112rtm and I ran the installer. 

However, after this I started getting a Fake McAfee WebAdvisory Pop-up notification every time I started my default browser (Google Chrome). When I started Microsoft Edge there was no problem. I scanned the exe-file and my whole system for viruses (Avira Antivir) but it found nothing thankfully. Still the problem persisted. 

The problem was resolved only after I had completely removed Google Chrome and then reinstalled it from scratch.

Has anyone else had a similar problem or experience?

**Comment by wyrdfish** [comment]
No, this release has been available for several years and is not infected. You even scanned it yourself. You must have been compromised elsewhere.
